client/tokenized-payment-request/index.js

```diff
@@ -0,0 +1,78 @@
+/* global jQuery, wcpayPaymentRequestParams */
+/**
+ * External dependencies
+ */
+import { doAction } from '@wordpress/hooks';
+
+/**
+ * Internal dependencies
+ */
+import WCPayAPI from '../checkout/api';
+import PaymentRequestCartApi from './cart-api';
+import WcpayPaymentRequest from './payment-request';
+import paymentRequestButtonUi from './button-ui';
+import './compatibility/wc-deposits';
+import './compatibility/wc-product-variations';
+
+import '../checkout/express-checkout-buttons.scss';
+
+jQuery( ( $ ) => {
+	// Don't load if blocks checkout is being loaded.
+	if (
+		wcpayPaymentRequestParams.has_block &&
+		wcpayPaymentRequestParams.button_context !== 'pay_for_order'
+	) {
+		return;
+	}
+
+	const publishableKey = wcpayPaymentRequestParams.stripe.publishableKey;
+
+	if ( ! publishableKey ) {
+		// If no configuration is present, we can't do anything.
+		return;
+	}
+
+	// initializing the UI's container.
+	paymentRequestButtonUi.init( {
+		$container: jQuery( '#wcpay-payment-request-button' ),
```

Comment:
```suggestion
		$container: $( '#wcpay-payment-request-button' ),
```
Not a huge difference, but why not?

Reply:
I don't really have a reason 😄